Week3Tasks/Week3/src/com/week3/BankApplication.java

```diff
@@ -0,0 +1,8 @@
+package com.week3;
+import java.util.Scanner;
+public class BankApplication {
```

Comment:
inside the bank application main class , you should test all the operations like adding new account also save account and see the balance ...etc